### ISSUE TITLE
Batch page archive doesn't require confirmation

### ISSUE THREAD

**Issue by maxime-rainville** [silverstripe/silverstripe-cms#2662]
https://youtu.be/ypicSZjN_k4

# Steps
- Access the CMS
- Click "Batch actions" buttion
- Select multiple pages
- Choose the "Publish" action and click GO
  - You have to accept a confirmation medal
- Choose a few more pages
- Select the "Unpublish and Archive" action and click GO
  - No confirmation is shown

It seems a bit weird that the more destructive action doesn't require a confirmation while the least destructive one does.

@silverstripeux

**Comment by clarkepaul** [comment]
Agree, it's more likely you'd need that additional confirmation on destructive things rather than constructive actions. In this case, I'd recommend having the confirmation on all batch actions for consistency and that we are dealing with an unknown amount of content. 

In saying that we should probably also check file batch actions and GridField actions for consistency too.
